src/views/BrowseInventory/BrowseInventory.js

```diff
@@ -80,10 +93,35 @@ const BrowseInventory = () => {
   const searchableIndexesPlaceholder = intl.formatMessage({ id: 'ui-inventory.browse.searchableIndexesPlaceholder' });
   const isResetButtonDisabled = !location.search && !searchQuery;
 
-  const formattedSearchableIndexes = browseInstanceIndexes.map(({ label, ...rest }) => ({
-    label: intl.formatMessage({ id: label }),
-    ...rest,
-  }));
+  const searchableOptions = browseInstanceIndexes.map((searchableIndex, index) => {
+    if (searchableIndex.subIndexes) {
+      return (
+        <optgroup
+          key={index}
+          label={intl.formatMessage({ id: searchableIndex.label })}
+          className={css.optgroup}
+        >
+          {searchableIndex.subIndexes.map((subOption, i) => (
+            <option
+              key={i}
+              value={subOption.value}
+            >
+              {intl.formatMessage({ id: subOption.label })}
+            </option>
+          ))}
+        </optgroup>
+      );
+    }
+
+    return (
+      <option
+        key={index}
```

Comment:
Let's use value as key

Reply:
I'll change the `index` to value, but I think it's unnecessary because it's not possible to change the order of these options. The `index` always will be the same.